federation/api/install/install.go

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
```

Comment:
as we will add more k8s objects here, I believe keep the same directory structure would help for future work.
Although I still fell unnecessary of registering only service object for the efforts it took and will take.